app/views/content-design/style-guide/index.html

```diff
@@ -68,38 +68,40 @@
 
 {% markdown %}
 
- The style guide includes words and language that differ from the <a class="govuk-link" href="https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style">Government Digital Service (GDS) style guide </a> that are used across multiple teams or areas in DfE.
+ Always use the <a class="govuk-link" href="https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style">Government Digital Service (GDS) style guide</a> first. 
+ 
+ This style guide includes words that differ the GDS style guide and are used across multiple teams or areas in DfE.
 
-It's a guide, not a set of rules. If your service uses niche language, or research shows your users say something else, use the words your users understand.  
+It is a guide, not a set of rules. If your service uses niche language, or research shows your users say something else, use the words your users understand.  
 
-If a word you use isn't there, start a conversation in the <a class="govuk-link" href="https://ukgovernmentdfe.slack.com/archives/C6L4J5DK6"> DfE #contentdesign Slack channel</a>, or email <a class="govuk-link" href="mailto:design.ops@education.gov.uk">design.ops@education.gov.uk</a>.
+If a word you use isn't here, <a class="govuk-link" href="https://ukgovernmentdfe.slack.com/archives/C6L4J5DK6">start a conversation in the DfE #contentdesign Slack channel</a>, or email <a class="govuk-link" href="mailto:design.ops@education.gov.uk">design.ops@education.gov.uk</a> and suggest it is added.
 
-You also can contribute to the DfE style guide on <a class="govuk-link" href="https://github.com/DFE-Digital/design/issues/111">GitHub</a>.
+You can also <a class="govuk-link" href="https://github.com/DFE-Digital/design/issues/111">contribute to the DfE style guide on GitHub</a>.
 
 ## A
 
 ### academic year 
 
-Use months rather than the academic year when communicating with the public. 
+Use months rather than the academic year when communicating with the public, especially if the content is aimed at an international audience. 
 
-For example, teacher training lasts around 9 months, usually starting in September. 
+For example, "Teacher training lasts around 9 months, usually starting in September." 
 
-Instead of, teacher training lasts one academic year. 
+Instead of, "Teacher training lasts one academic year."
 
-Use a forward slash to separate the two years within a single academic year, for example,'in the academic year 2016/17' 
+If you need to write about a specific academic year, <a class="govuk-link" href="https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#dates">follow GDS guidance for dates and date ranges</a>.
```

Comment:
Just a note that you can use markdown for links here (as well as HTML), if that's easier.

```suggestion
If you need to write about a specific academic year, [follow GDS guidance for dates and date ranges](https://www.gov.uk/guidance/style-guide/a-to-z-of-gov-uk-style#dates).
```